src/pivot-table/components/shared-styles.ts

```diff
@@ -90,3 +91,27 @@ export const getTotalCellDividerStyle = ({
 
   return style;
 };
+
+interface StyledLeftGridProps {
+  leftGridWidth: number;
+}
+
+export const StyledLeftGrid = styled("div", {
+  shouldForwardProp: (prop: string) => !["leftGridWidth"].includes(prop),
+})(({ leftGridWidth }: StyledLeftGridProps) => ({
+  width: leftGridWidth,
```

Comment:
Not sure this is a great idea. I think that every time the chart is re-size or a column is re-size a new css class will be created by mui

Reply:
I interpret that as a new (re)render cycle and every thing will start from scratch which is already happening for/in other (styled) components as well, do you see any problem with that?

Reply:
When testing re-sizing the chart, it generated a new inline style sheets on every re-size event. I needed to refresh the browser for the style sheets to get purged.